src/server/pfs/server/option.go

```diff
@@ -0,0 +1,22 @@
+package server
+
+import "github.com/pachyderm/pachyderm/v2/src/internal/storage/fileset"
+
+type SourceOption func(*sourceConfig)
+
+type sourceConfig struct {
+	full   bool
+	filter func(fileset.FileSet) fileset.FileSet
+}
+
+func WithFull() SourceOption {
```

Comment:
What do either of these do? Add comments to the constructors

Reply:
Sure, I'm surprised lint didn't complain about this.

Reply:
Yeah I think linting broke, or this was disabled.  Comments on exports was one of the more important rules in my opinion.